pkg/sfu/downtrack.go

```diff
@@ -647,13 +647,18 @@ func (d *DownTrack) OnPacketSent(fn func(dt *DownTrack, size int)) {
 	d.onPacketSent = append(d.onPacketSent, fn)
 }
 
-func (d *DownTrack) AdjustAllocation(availableChannelCapacity uint64) (uint64, uint64) {
+func (d *DownTrack) AdjustAllocation(availableChannelCapacity uint64) (isPausing, isResuming bool, bandwidthRequested, optimalBandwidthNeeded uint64) {
```

Comment:
note for the future, it'd be great if there's a way to test this function.

Reply:
Yes, definitely. Need to come up with interfaces so that we can mock and test.